firstvoices/backend/models/characters.py

```diff
@@ -37,7 +37,7 @@ class Meta:
         rules_permissions = {
             "view": predicates.has_visible_site,
             "add": predicates.is_superadmin,
-            "change": predicates.is_superadmin,
+            "change": predicates.is_at_least_assistant_or_super,
```

Comment:
Everything to do with characters is restricted to only language admins, so this should probably be `is_language_admin_or_super`